packages/components/addon/components/hds/tabs/panel.js

```diff
@@ -15,11 +15,15 @@ export default class HdsTabsIndexComponent extends Component {
 
   @cached
   get nodeIndex() {
-    return this.args.panelIds.indexOf(this.panelId);
+    return this.args.panelIds
+      ? this.args.panelIds.indexOf(this.panelId)
+      : undefined;
```

Comment:
I have added this logic so the `Hds::Tabs::Panel` can be used as standalone component (without a `Tabs` parent).